tools/binlog/tidb-binlog-local.md

```diff
@@ -0,0 +1,287 @@
+---
+title: TiDB-Binlog Local 部署方案
+category: tools
+---
+
+# TiDB-Binlog Local 部署方案
+
+## TiDB-Binlog-local 简介
+
+TiDB-Binlog 用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+
+TiDB-Binlog 支持以下功能场景:
+
+* **数据同步**：同步 TiDB 集群数据到其他数据库。
+* **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复。
+
+## TiDB-Binlog-local 架构
+
+首先介绍 TiDB-Binlog 的整体架构。
+
+![TiDB-Binlog 架构](../../media/architecture.jpeg)
+
+TiDB-Binlog 集群主要分为两个组件：
+
+#### Pump
+
+Pump 是一个守护进程，在每个 TiDB 的主机上后台运行。他的主要功能是实时记录 TiDB 产生的 Binlog 并顺序写入磁盘文件
+
+#### Drainer
+
+Drainer 从各个 Pump 节点收集 Binlog，并按照在 TiDB 中事务的提交顺序转化为指定数据库兼容的 SQL 语句，最后同步到目的数据库或者写到顺序文件
+
+## TiDB-Binlog-local 安装
+
+### TiDB-Binlog-local 下载
+
+TiDB-Binlog 包含在 tidb-enterprise-tools 安装包中，可[在此下载](../tools/download.md)。
```

Comment:
seems invalid link.

Reply:
Fixed in https://github.com/pingcap/docs-cn/pull/1264/commits/5fb13216ccc83d5f74fa9ceba46b02a3f7b538d8